### ISSUE TITLE
questions about the BED file

### ISSUE THREAD

**Issue by gengyuncong** [remap-cisreg/remap-pipeline#2]
Hi, 

I downloaded the BED file for all peaks data. The fifth column seems to be a score. What does that represent? How is that calculates? 

Best,
Yuncong

**Comment by benoitballester** [comment]
Hi Yuncong, 

Thanks for your interest in ReMap data. 
Form what I understand you have downloaded the `all peaks` BED file from the ReMap [download page](http://remap.univ-amu.fr/download_page).

This BED file follows the recommended UCSC [BED9 format](https://genome.ucsc.edu/FAQ/FAQformat.html#format1).

Below is a recap (copy/paste) of what BED columns are:
**chrom** - The name of the chromosome (e.g. chr3, chrY, chr2_random) or scaffold (e.g. scaffold10671).

**chromStart** - The starting position of the feature in the chromosome or scaffold. T

**chromEnd** - The ending position of the feature in the chromosome or scaffold. 

**name** - Defines the name of the BED line. 

**score** - A score between 0 and 1000. If the track line useScore attribute is set to 1 for this annotation data set, the score value will determine the level of gray in which this feature is displayed (higher numbers = darker gray). This table shows the Genome Browser's translation of BED score values into shades of gray:
shade	 	 	 	 	 	 	 	 	 

**strand** - Defines the strand. Either "." (=no strand) or "+" or "-".

**thickStart** - The starting position at which the feature is drawn thickly (for example, the start codon in gene displays). When there is no thick part, thickStart and thickEnd are usually set to the chromStart position.

**thickEnd** - The ending position at which the feature is drawn thickly (for example the stop codon in gene displays).

**itemRgb** - An RGB value of the form R,G,B (e.g. 255,0,0). 


Regarding the score columns, I would not take it into account, as it contains scores from the ReMap2020 update (MACS q-value) and scores from the ReMap2018 catalogue which were set to 0. Indeed, as this is a compendium of independent MACS peaks calls, the scores from two different datasets are not really comparable. So in 2018 we decided to set them to 0, but kept the score for 2020 new data (I hope this is clear).
We need to homogenise this for the ReMap 2022 update. Either we set all of the scores to 0, or we put the original MACS q-values.

Hope this helps.,

Ben
